tests/lib/tools/suite/fs-state/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: smoke test for the fs-state tool
+
+execute: |
+    # Check help
+    "$TESTSTOOLS"/fs-state -h | MATCH "usage: fs-state wait-for-file"
+    "$TESTSTOOLS"/fs-state --help | MATCH "usage: fs-state wait-for-file"
+    "$TESTSTOOLS"/fs-state | MATCH "usage: fs-state wait-for-file"
+
+    # Check non existing directory
+    "$TESTSTOOLS"/fs-state mock-dir mytestdir
+    test -d mytestdir
+    test -f mytestdir.fake
+    not test -d mytestdir.back
+
+    "$TESTSTOOLS"/fs-state restore-dir mytestdir
+    not test -d mytestdir
```

Comment:
Could you also check that `.fake.` is gone please.

Reply:
done